generate-package-notes.sh

```diff
@@ -148,6 +163,26 @@ parse_options() {
         shift
     done
 
+    # Parse at the end, so that --root can be used in any position
+    if [ "${cpe}" = "auto" ]; then
+        if [ -r "${root}/usr/lib/system-release-cpe" ]; then
+            cpe="$(cat "${root}/usr/lib/system-release-cpe")"
+        elif [ -r "${root}/etc/os-release" ]; then
+            # shellcheck disable=SC1090 disable=SC1091
+            cpe="$(. "${root}/etc/os-release" && echo "${CPE_NAME}")"
+        elif [ -r "${root}/usr/lib/os-release" ]; then
+            # shellcheck disable=SC1090 disable=SC1091
+            cpe="$(. "${root}/usr/lib/os-release" && echo "${CPE_NAME}")"
+        fi
+        if [ -z "${cpe}" ]; then
+            printf 'ERROR: --auto-cpe %s but cannot read %s/usr/lib/system-release-cpe or parse CPE_NAME from %s/etc/os-release or %s/usr/lib/os-release.\n' "${2}" "${root}" "${root}" "${root}" >&2
```

Comment:
```suggestion
            printf 'ERROR: --cpe=%s but cannot read %s/usr/lib/system-release-cpe or parse CPE_NAME from %s/etc/os-release or %s/usr/lib/os-release.\n' "${2}" "${root}" "${root}" "${root}" >&2
```

Also, I'm not sure if the `"${2}"` is right here, since it won't be the original value set for the `--cpe` option, as it's shifted away during parsing (or maybe I'm misunderstanding the original intent). So, again maybe, using just `--cpe=auto` in the message should be enough?